CONTRIBUTING.md

```diff
@@ -0,0 +1,229 @@
+Contributing to the Project
+===================================
+
+Thank you for your interest in contributing to the MinimumViableDataspace!
+
+## Table of Contents
+
+* [Project Description](#project-description)
+* [Code Of Conduct](#code-of-conduct)
+* [Eclipse Contributor Agreement](#eclipse-contributor-agreement)
+* [How to Contribute](#how-to-contribute)
+  * [Discuss](#discuss)
+  * [Create an Issue](#create-an-issue)
+  * [Submit a Pull Request](#submit-a-pull-request)
+  * [Report on Flaky Tests](#report-on-flaky-tests)
+* [Project and Milestone Planning](#project-and-milestone-planning)
+  * [Milestones](#milestones)
+  * [Projects](#projects)
+* [Contact Us](#contact-us)
+
+## Project Description
+
+See [README.md](README.md) for a comprehensive project description.
+
+## Code Of Conduct
+
+See the [Eclipse Code Of Conduct](https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php).
+
+## Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project, you need to create and electronically sign
+a [Eclipse Contributor Agreement (ECA)](http://www.eclipse.org/legal/ecafaq.php):
+
+1. Log in to the [Eclipse foundation website](https://accounts.eclipse.org/user/login/). You will 
+   need to create an account within the Eclipse Foundation if you have not already done so.
+2. Click on "Eclipse ECA", and complete the form.
+
+Be sure to use the same email address in your Eclipse Account that you intend to use when you commit 
+to GitHub.
+
+## How to Contribute
+
+### Discuss
+
+If you want to share an idea to further enhance
+the project or discuss potential use cases, please feel free to create a discussion at the
+[GitHub Discussions page](https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/discussions).
```

Comment:
@paullatzelsperger these discussions tag do not exist yet, do we want to enable them in this repo?